File: 80-remove-duplicates-from-sorted-array-ii/80-remove-duplicates-from-sorted-array-ii.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums) - 1
        if n < 2:
            return n+1
        k = 1
        index = 0
        for index in range(n+1):
            if index > 1:
                if nums[index] != nums[k-1]:
                    k = k + 1
                    nums[k] = nums[index]
        return k + 1

        

File: 80-remove-duplicates-from-sorted-array-ii/80-remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int k = 0;
        for(int i = 0; i < nums.length; i++){
            if(k >= 2 && nums[i] > nums[k-2]){
                int temp = nums[k];
                nums[k] = nums[i];
                nums[i] = temp;
                k++;
            }else if (k < 2){
                k++;
            }
        }
        return k;
    }
}